lamindb/dev/db/_add.py

```diff
@@ -38,22 +48,34 @@ def add(  # type: ignore  # no support of different naming of args across overlo
     Example:
 
     >>> # insert a new record
-    >>> db.add(wetlab.experiment(name="My test", biometa_id=test_id))
+    >>> ln.add(wetlab.Experiment(name="My test", biometa_id=test_id))
     >>> # update an existing record
-    >>> experiment = ln.select(wetlab.experiment, id=experiment_id).one()
+    >>> experiment = ln.select(wetlab.Experiment, id=experiment_id).one()
     >>> experiment.name = "New name"
-    >>> db.add(experiment)
+    >>> ln.add(experiment)
+    >>> # check existence before inserting a new record
```

Comment:
How about `# add a record for fields if not yet exists`?

Reply:
ok

Reply:
by fields?